### DISCUSSION TITLE
Looking for headless version for server, command line only

### DISCUSSION THREAD

**Discussion by falves1** [abraunegg/onedrive#2045] [General]
I tried to install this on a server, via ssh, and it started installing a graphic environment, so I stopped it.
How do I mount a onedrive link on bash only?

**Comment by abraunegg** [comment]
@falves1 
> I tried to install this on a server, via ssh, and it started installing a graphic environment, so I stopped it.

The client has *zero* requirement for any graphical environment .... there should be *zero* graphic requirement being installed.

What instructions were you following?

> How do I mount a onedrive link on bash only?

You install the client, then configure the client to access your OneDrive account - no more to do.
